test/evergreen.yml

```diff
@@ -977,7 +976,6 @@ tasks:
       - func: "fetch artifacts"
       - command: shell.exec
         params:
-          working_dir: "wiredtiger/build_posix/"
```

Comment:
You should leave the working_dir and update line 983

Reply:
Yeah but the EXT_PATH in the api_const.h is `../../ext` which is good for users running it but breaks given the working dir of `wiredtiger/build_posix/`